docs/data/date-pickers/fields/LocalizedDateField.tsx

```diff
@@ -1,17 +1,17 @@
 import * as React from 'react';
 import dayjs, { Dayjs } from 'dayjs';
+import 'dayjs/locale/fr';
+import { DemoContainer } from 'docsx/src/modules/components/DemoContainer';
 import { LocalizationProvider } from '@mui/x-date-pickers';
 import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
 import { Unstable_DateField as DateField } from '@mui/x-date-pickers/DateField';
-import Stack from '@mui/material/Stack';
-import 'dayjs/locale/fr';
 
 export default function LocalizedDateField() {
   const [value, setValue] = React.useState<Dayjs | null>(dayjs('2022-04-07'));
 
   return (
     <LocalizationProvider dateAdapter={AdapterDayjs} adapterLocale="fr">
-      <Stack spacing={2} sx={(theme) => ({ width: theme.spacing(48) })}>
+      <DemoContainer>
```

Comment:
Is this demo used anywhere? I can't seem to find it 🙈

Reply:
Good catch, removed